ipynb/examples/android/benchmarks/Android_Jankbench.ipynb

```diff
@@ -399,7 +399,8 @@
     "def overall_statistics(df):\n",
     "    byname_test = df.groupby(['name','test']).total_duration.describe(percentiles=[0.9, 0.95, 0.99])\n",
     "    stats = pd.DataFrame(byname_test)\n",
-    "    stats = stats.unstack()\n",
+    "    # If your version of Pandas is older than 0.20.1, you may want to uncomment this line:\n",
+    "    # stats = stats.unstack()\n",
```

Comment:
Here:
   https://github.com/ARM-software/lisa/blob/master/libs/utils/wa_results_collector.py#L707
we have an example of how we can make this code more "generic"

Reply:
Updated